$OpenBSD$

SPDX-License-Identifier: AGPL-3.0-only

Keep generated migration SQL column definitions consistently indented.

Index: src/Simplex/Messaging/Agent/Store/SQLite/Migrations/agent_schema.sql
--- src/Simplex/Messaging/Agent/Store/SQLite/Migrations/agent_schema.sql.orig
+++ src/Simplex/Messaging/Agent/Store/SQLite/Migrations/agent_schema.sql
@@ -203,10 +204,10 @@
   ntf_port TEXT NOT NULL,
   tkn_id BLOB, -- token ID assigned by notifications server
   tkn_pub_key BLOB NOT NULL, -- client's public key to verify token commands(used by server, for repeat registraions)
-tkn_priv_key BLOB NOT NULL, -- client's private key to sign token commands
-tkn_pub_dh_key BLOB NOT NULL, -- client's public DH key(for repeat registraions)
-tkn_priv_dh_key BLOB NOT NULL, -- client's private DH key(for repeat registraions)
-tkn_dh_secret BLOB, -- DH secret for e2e encryption of notifications
+  tkn_priv_key BLOB NOT NULL, -- client's private key to sign token commands
+  tkn_pub_dh_key BLOB NOT NULL, -- client's public DH key(for repeat registraions)
+  tkn_priv_dh_key BLOB NOT NULL, -- client's private DH key(for repeat registraions)
+  tkn_dh_secret BLOB, -- DH secret for e2e encryption of notifications
   tkn_status TEXT NOT NULL,
   tkn_action BLOB,
   created_at TEXT NOT NULL DEFAULT(datetime('now')),
@@ -423,8 +424,8 @@
   ntf_key_hash BLOB NOT NULL,
   tkn_id BLOB NOT NULL, -- token ID assigned by notifications server
   tkn_priv_key BLOB NOT NULL, -- client's private key to sign token commands,
-del_failed INTEGER DEFAULT 0,
-created_at TEXT NOT NULL DEFAULT(datetime('now'))
+  del_failed INTEGER DEFAULT 0,
+  created_at TEXT NOT NULL DEFAULT(datetime('now'))
 ) STRICT;
 CREATE TABLE snd_message_bodies(
   snd_message_body_id INTEGER PRIMARY KEY,

